README.md

```diff
@@ -21,6 +21,26 @@ npm install --save @feedzai/brushable-histogram
 
 Note that by default no style is included. If you want you can use the css at `@feedzai/brushable-histogram/lib/css/brushable-histogram.css` to get the same style as in the examples.
 
+## Installing dependencies
```

Comment:
Actually since react is also on the dev dependencies we don't need to install the peer dependencies explicitly, right?

Reply:
It is a dev dependency in Histogram. In Genome or any other project, we want to have that as a dependency and if we don't have `react` installed this will be the way to do that.

Reply:
Ah, ok. I see. I think it's not common to have this because if you are using this package you will already have react installed, right?

Reply:
True. I was thinking about the worst scenario :sweat_smile:

We can assume that we already have react installed and the peerDependency just stay there to make sure that we have a supported version.

Reply:
Yes, I think that makes more sense in this case

Reply:
:heavy_check_mark: